geps/gep-1619/index.md

```diff
@@ -926,13 +920,14 @@ spec:
     type: Cookie
 ```
 
-A potentially unexpected situation occurs when:
+In this traffic splitting scenario within a single route rule, this GEP leaves the decision to the implementation. An
+implementation MAY choose to:
```

Comment:
You updated the wording to `MUST` above. That makes sense, as they MUST choose an option. Should we do that here?

```suggestion
implementation MUST choose one of the following:
```